batch-nft-reveal/app/components/collection/reveal-info/Revealinfo.tsx

```diff
@@ -0,0 +1,37 @@
+import { Container, Heading } from '@chakra-ui/react'
+import { BigNumber } from 'ethers'
+import { useNextRevealAmount } from '../../../hooks/useNextRevealAmount'
+import { useNextRevealInterval } from '../../../hooks/useNextRevealInterval'
+import { AddressProp } from '../../../types/AddressProp'
+import { NextReveal } from '../reveal-info/NextReveal'
+
+export const RevealInfo = (props: AddressProp): JSX.Element => {
+  const { contractAddress } = props
+
+  const nextRevealBatchSize = useNextRevealAmount(contractAddress)
+  const nextRevealInterval = useNextRevealInterval(contractAddress)
+
+  let pending = false
+
+  if (
+    nextRevealBatchSize <= BigNumber.from(0) ||
+    nextRevealInterval < BigNumber.from(Date.now())
```

Comment:
Please check how to properly compare `BigNumber` objects: https://mikemcl.github.io/bignumber.js/#lte